packages/zowe-explorer/i18n/sample/package.i18n.json

```diff
@@ -77,6 +77,7 @@
   "zowe.commands.history": "Toggle if Commands persist locally",
   "zowe.automaticProfileValidation": "Allow automatic validation of profiles.",
   "zowe.settings.version": "Track if migration has been processed",
+  "zowe.security.enableSecureCredentials": "Store username and password for z/OS as clear text in Zowe team config profiles if disabled",
```

Comment:
I think better naming is `zowe.security.disableSecureCredentials`

because it's enabled by default and nothing needs to be done. Option enableSecureCredentials makes it seems like secure credentials need to be enabled, like it's an optional/experimental feature. In reality it's the default option, and if user wants to disable it, they should be tweaking `disableSecureCredentials` setting.

Reply:
unchecking the setting box the value changes to false in the settings.json so this request would not make much sense.  possibly have it as `secureCredentialsEnabled` so when unticked that will = false

Reply:
I updated to secureCredentialsEnabled